### DISCUSSION TITLE
lldp in sonic can't take the 802.1p infomations to the remote.

### DISCUSSION THREAD

**Discussion by coolkidBigEye** [lldpd/lldpd#454] [Q&A]
Hi, ask a question about lldp. 
I use the version lldpd_0.9.6.
![image](https://user-images.githubusercontent.com/72586704/123726027-c7f5e200-d8c1-11eb-81b6-27bf757afa7e.png)
I add the Ethernet12 in Vlan10, but the lldp packet does not include the vlan info.
![image](https://user-images.githubusercontent.com/72586704/123726673-f9bb7880-d8c2-11eb-9891-7a7f613c6f25.png)
dump the lldp packet as follows:
root@sonic:/sys/devices/virtual/net/Ethernet12/brport/bridge/upper_Vlan10# tcpdump -s0 -vv -pni Ethernet12 ether dst 01:80:c2:00:00:0e
tcpdump: listening on Ethernet12, link-type EN10MB (Ethernet), capture size 262144 bytes
04:08:17.767765 LLDP, length 190
        Chassis ID TLV (1), length 7
          Subtype MAC address (4): 74:fe:48:3e:73:bf
          0x0000:  0474 fe48 3e73 bf
        Port ID TLV (2), length 12
          Subtype Local (7): 100GE1/1/12
          0x0000:  0731 3030 4745 312f 312f 3132
        Time to Live TLV (3), length 2: TTL 120s
          0x0000:  0078
        System Name TLV (5), length 2: 28
          0x0000:  3238
        System Description TLV (6), length 97
          Debian GNU/Linux 10 (buster) Linux 4.19.0-9-2-amd64 #1 SMP Wed, 29 Apr 2020 11:38:41 +0200 x86_64
          0x0000:  4465 6269 616e 2047 4e55 2f4c 696e 7578
          0x0010:  2031 3020 2862 7573 7465 7229 204c 696e
          0x0020:  7578 2034 2e31 392e 302d 392d 322d 616d
          0x0030:  6436 3420 2331 2053 4d50 2057 6564 2c20
          0x0040:  3239 2041 7072 2032 3032 3020 3131 3a33
          0x0050:  383a 3431 202b 3032 3030 2078 3836 5f36
          0x0060:  34
        System Capabilities TLV (7), length 4
          System  Capabilities [Bridge, WLAN AP, Router, Station Only] (0x009c)
          Enabled Capabilities [Bridge, Router] (0x0014)
          0x0000:  009c 0014
        Management Address TLV (8), length 12
          Management Address length 5, AFI IPv4 (1): 125.78.29.28
          Unknown Interface Numbering (1): 0
          0x0000:  0501 7d4e 1d1c 0100 0000 0000
        Port Description TLV (4), length 10: Ethernet12
          0x0000:  4574 6865 726e 6574 3132
        Organization specific TLV (127), length 9: OUI IEEE 802.3 Private (0x00120f)
          Link aggregation Subtype (3)
            aggregation status [supported], aggregation port ID 0
          0x0000:  0012 0f03 0100 0000 00
        Organization specific TLV (127), length 9: OUI IEEE 802.3 Private (0x00120f)
          MAC/PHY configuration/status Subtype (1)
            autonegotiation [none] (0x00)
            PMD autoneg capability [unknown] (0x8000)
            MAU type 10BASET fdx (0x000b)
          0x0000:  0012 0f01 0080 0000 0b
        End TLV (0), length 0

can someone tell me the reason?

**Comment by vincentbernat** [comment]
VLAN-aware bridge support has been added in 1.0.5. Prior versions only know about VLAN interfaces.

**Reply by coolkidBigEye** [reply]
I see，could you give me the mr, so I can make the patch to test.

**Reply by vincentbernat** [reply]
There are several of them:

 - 011056b3a7143458a37d201cba52633d3c7a5dd8
 - e512429c88d2dfa7d37fa39b2c6735d0c8f1042f
 - 385e5e4cf3a74e1d56947418db49911640cfc4a9

**Reply by coolkidBigEye** [reply]
thks. 
By the way, I need to know this modify whether something is relied on .  My test enviroment is debian 10.
